.github/workflows/ci.yml

```diff
@@ -75,22 +76,26 @@ jobs:
           - misc_part2
           - performance_specializations_part1
           - performance_specializations_part2
-          # - mpi # moved to ci_problematic.yml
+          - mpi
           - threaded
         include:
           - version: '1.8'
+            os: ubuntu-latest
+            arch: x64
+            trixi_test: threaded_legacy
```

Comment:
What exactly are we trying to achieve with this test? If we only want to ensure that we do not accidentally break backward compatibility to Julia v1.8 in a major way, this should be sufficient. However, to really claim that we support both v1.8 and v1.9, we should probably extend these tests also to macOS and Windows, don't you think?

Reply:
We don't do anything OS-specific inside of Trixi.jl, so I thought this would be enough. If you want to burn more CI time, I can add the additional jobs you mentioned.

Reply:
Right. Let's leave it like this for now then. We can always add it back if we have users complaining about weird errors that only occur with v1.8.